pkg/resource/user/model.go

```diff
@@ -5,9 +5,10 @@ import (
 )
 
 type User struct {
-	ClusterName               types.String `tfsdk:"cluster_name"`
-	ID                        types.String `tfsdk:"id"`
-	Name                      types.String `tfsdk:"name"`
-	PasswordSha256Hash        types.String `tfsdk:"password_sha256_hash_wo"`
-	PasswordSha256HashVersion types.Int32  `tfsdk:"password_sha256_hash_wo_version"`
+	ClusterName                   types.String `tfsdk:"cluster_name"`
+	ID                            types.String `tfsdk:"id"`
+	Name                          types.String `tfsdk:"name"`
+	PasswordHash                  types.String `tfsdk:"password_hash"`
```

Comment:
```suggestion
	PasswordHash                  types.String `tfsdk:"password_sha256_hash"`
```
For consistency, what do you think?